macOS support is not working

### I have searched and made sure there are no existing issues for the issue I am filing

- [X] I have searched the existing issues

### Description

I am on an M1 MacBook Pro.  

I create a new new project using `ti create`.  

I then run `ti build -p ios -T macos`.

I get the following error...

```
[TRACE]
[TRACE] LinkStoryboards (in target 'AnyPlatform' from project 'AnyPlatform')
[TRACE]     cd /Users/Matthew/Titanium/_Playground/AnyPlatform/build/iphone
[TRACE]     /Applications/Xcode.app/Contents/Developer/usr/bin/ibtool --errors --warnings --notices --platform macosx --module AnyPlatform --target-device ipad --minimum-deployment-target 13.1 --output-format human-readable-text --link /Users/Matthew/Titanium/_Playground/AnyPlatform/build/iphone/build/Products/Debug-maccatalyst/AnyPlatform.app/Contents/Resources /Users/Matthew/Titanium/_Playground/AnyPlatform/build/iphone/build/Intermediates/AnyPlatform.build/Debug-maccatalyst/AnyPlatform.build/LaunchScreen.storyboardc
[ERROR] --- xcodebuild: WARNING: Using the first of multiple matching destinations:
[ERROR] { platform:macOS, arch:arm64, variant:Mac Catalyst, id:00006001-001210200A02801E }
[ERROR] { platform:macOS, arch:x86_64, variant:Mac Catalyst, id:00006001-001210200A02801E }
[ERROR] { platform:macOS, arch:arm64, variant:Designed for [iPad,iPhone], id:00006001-001210200A02801E }
[ERROR] { platform:iOS, id:dvtdevice-DVTiPhonePlaceholder-iphoneos:placeholder, name:Any iOS Device }
[ERROR] { platform:iOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-iphonesimulator:placeholder, name:Any iOS Simulator Device }
[ERROR] { platform:macOS, variant:Mac Catalyst, name:Any Mac }
[ERROR] { platform:iOS Simulator, id:FB7CF4EE-69A4-4243-A364-B9E25C721ED8, OS:13.7, name:iPad (7th generation) }
[ERROR] { platform:iOS Simulator, id:F3BB5D71-D687-4AF1-9119-4B9F32CA164A, OS:14.0.1, name:iPad (8th generation) }
[ERROR] { platform:iOS Simulator, id:6337A94E-05F9-420E-8EC5-EB636E502281, OS:16.2, name:iPad (9th generation) }
[ERROR] { platform:iOS Simulator, id:67FF7672-5129-4B81-BEC7-A1869E52FA84, OS:16.2, name:iPad (10th generation) }
[ERROR] { platform:iOS Simulator, id:117B5DD8-88DB-46FF-A51F-1CB357BD831B, OS:13.7, name:iPad Air (3rd generation) }
[ERROR] { platform:iOS Simulator, id:B69051CA-5C1E-45F4-9B2E-597BF69A20C1, OS:14.0.1, name:iPad Air (4th generation) }
[ERROR] { platform:iOS Simulator, id:6B436BC5-5D22-4C0E-BE4D-2A9E65DC408A, OS:16.2, name:iPad Air (5th generation) }
[ERROR] { platform:iOS Simulator, id:8B1F1A06-558F-466F-A5F9-2460F0DE2BC8, OS:13.7, name:iPad Pro (9.7-inch) }
[ERROR] { platform:iOS Simulator, id:1B0F05D4-E3D3-4F4F-A3B7-42A82338F0F6, OS:14.0.1, name:iPad Pro (9.7-inch) }
[ERROR] { platform:iOS Simulator, id:19C87224-7DA5-42CE-84ED-A391E2961351, OS:13.7, name:iPad Pro (11-inch) (2nd generation) }
[ERROR] { platform:iOS Simulator, id:B2255D47-265F-4ADE-8179-9B749A82830D, OS:14.0.1, name:iPad Pro (11-inch) (2nd generation) }
[ERROR] { platform:iOS Simulator, id:E96DF372-22FE-41CA-8FA1-A3A81E828CA3, OS:16.2, name:iPad Pro (11-inch) (3rd generation) }
[ERROR] { platform:iOS Simulator, id:07185221-FB1C-4BB9-A227-BECDBB8D88F5, OS:16.2, name:iPad Pro (11-inch) (4th generation) }
[ERROR] { platform:iOS Simulator, id:4820E46D-31F1-4034-A40F-ADE2DE9A10E2, OS:16.2, name:iPad Pro (12.9-inch) (2nd generation) }
[ERROR] { platform:iOS Simulator, id:8D99AEC2-3574-44EB-BF1C-433BC46F6BC0, OS:13.7, name:iPad Pro (12.9-inch) (4th generation) }
[ERROR] { platform:iOS Simulator, id:E828B35D-3784-4666-8318-93E311001CA3, OS:14.0.1, name:iPad Pro (12.9-inch) (4th generation) }
[ERROR] { platform:iOS Simulator, id:3DB35094-A636-458A-AD77-6A197B240028, OS:16.2, name:iPad Pro (12.9-inch) (5th generation) }
[ERROR] { platform:iOS Simulator, id:3B03311B-E9BF-4C2D-8CB4-5366C42FD657, OS:16.2, name:iPad Pro (12.9-inch) (6th generation) }
[ERROR] { platform:iOS Simulator, id:368C1487-2D0A-46D3-B2A2-E44556004ED9, OS:16.2, name:iPad mini (6th generation) }
[ERROR] { platform:iOS Simulator, id:C182E4AE-0D41-48A6-AFD5-E94C82ACF3BB, OS:16.2, name:iPhone 8 Plus }
[ERROR] { platform:iOS Simulator, id:1250CB23-726E-4379-A72F-90D40D288150, OS:13.7, name:iPhone 11 }
[ERROR] { platform:iOS Simulator, id:DF89A504-7B54-4AC2-BFFD-5142699F8FBE, OS:13.7, name:iPhone 11 Pro }
[ERROR] { platform:iOS Simulator, id:D22D8C72-E329-4B24-826E-94448F7622C7, OS:13.7, name:iPhone 11 Pro Max }
[ERROR] { platform:iOS Simulator, id:EBFF6DFD-C1F0-4763-BB81-8F23C7C9DB2F, OS:14.0.1, name:iPhone 11 Pro Max }
[ERROR] { platform:iOS Simulator, id:7D06A0B0-00B9-48B2-A06A-8A71F45F2351, OS:16.2, name:iPhone 11 Pro Max }
[ERROR] { platform:iOS Simulator, id:A58FF29E-5ACA-4C80-9A3A-DB267A9B28CA, OS:16.2, name:iPhone 14 }
[ERROR] { platform:iOS Simulator, id:D0070ACC-583A-433D-9E7E-B261D09E3FC9, OS:16.2, name:iPhone 14 Plus }
[ERROR] { platform:iOS Simulator, id:5B58CEE2-1392-46B8-848D-C632E917A092, OS:16.2, name:iPhone 14 Pro }
[ERROR] { platform:iOS Simulator, id:38F5166F-529D-4589-A08E-1BBCBAD51E4E, OS:16.2, name:iPhone 14 Pro Max }
[ERROR] { platform:iOS Simulator, id:E211DE98-7430-4EAE-BE09-A8A42E7AE03D, OS:13.7, name:iPhone SE (2nd generation) }
[ERROR] { platform:iOS Simulator, id:1E5B00DB-F6D6-42AF-9977-E707D91BDB96, OS:14.0.1, name:iPhone SE (2nd generation) }
[ERROR] { platform:iOS Simulator, id:435BA1CD-24AE-423E-889F-61AFE60A64E5, OS:16.2, name:iPhone SE (3rd generation) }
[ERROR] { platform:iOS Simulator, id:86229C6C-D4D0-45B5-9892-44BD3CF30C57, OS:14.0.1, name:iPod touch (7th generation) }
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEPLOYTYPE=development')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEPLOYTYPE=development')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than '__LOG__ID__=e3a2d346-efbb-4149-80b7-8af1aa46fa5d')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than '__LOG__ID__=e3a2d346-efbb-4149-80b7-8af1aa46fa5d')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEBUG=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEBUG=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'LAUNCHSCREEN_STORYBOARD=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'LAUNCHSCREEN_STORYBOARD=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEFAULT_BGCOLOR_RED=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEFAULT_BGCOLOR_RED=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEFAULT_BGCOLOR_GREEN=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEFAULT_BGCOLOR_GREEN=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEFAULT_BGCOLOR_BLUE=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEFAULT_BGCOLOR_BLUE=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DISABLE_TI_LOG_SERVER=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DISABLE_TI_LOG_SERVER=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'TI_VERSION=11.1.0')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'TI_VERSION=11.1.0')
[ERROR] ** BUILD FAILED **
[ERROR] The following build commands failed:
[ERROR]         SwiftEmitModule normal x86_64 Emitting\ module\ for\ AnyPlatform (in target 'AnyPlatform' from project 'AnyPlatform')
[ERROR] (1 failure)
```

### Expected Behavior

The project should build and open a simulator.

### Actual behavior

Crash on build.

### Reproducible sample

I create a new new project using `ti create`.  

I then run `ti build -p ios -T macos`.

### Steps to reproduce

I create a new new project using `ti create`.  

I then run `ti build -p ios -T macos`.

### Platform

_No response_

### SDK version you are using

12

### Alloy version you are using

_No response_

I hotpatched that file in my local 12.0.0.GA. Created a new project using the steps above. And now I get this error:

```
[TRACE] /Users/Matthew/Titanium/_Playground/Catalyst/build/iphone/Catalyst-Bridging-Header.h:6:9: note: in file included from /Users/Matthew/Titanium/_Playground/Catalyst/build/iphone/Catalyst-Bridging-Header.h:6:
[TRACE] #import "TiAppiOSActivityAttributesProxy.h"
[TRACE]         ^
[TRACE] /Users/Matthew/Titanium/_Playground/Catalyst/build/iphone/Classes/TiAppiOSActivityAttributesProxy.h:10:9: error: 'TitaniumKit/TitaniumKit.h' file not found
[TRACE] #import <TitaniumKit/TitaniumKit.h>
[TRACE]         ^
[TRACE] /Users/Matthew/Titanium/_Playground/Catalyst/build/iphone/Classes/TiAppiOSActivityAttributesProxy.h:10:9: note: did not find header 'TitaniumKit.h' in framework 'TitaniumKit' (loaded from '/Users/Matthew/Titanium/_Playground/Catalyst/build/iphone/build/Products/Debug-maccatalyst')
[TRACE] #import <TitaniumKit/TitaniumKit.h>
[TRACE]         ^
[TRACE] 1 error generated.
[TRACE] <unknown>:0: error: failed to emit precompiled header '/Users/Matthew/Titanium/_Playground/Catalyst/build/iphone/build/Intermediates/PrecompiledHeaders/Catalyst-Bridging-Header-swift_APM9P36QTNQN-clang_Q4UAL5UZ0VV2.pch' for bridging header '/Users/Matthew/Titanium/_Playground/Catalyst/build/iphone/Catalyst-Bridging-Header.h'
[TRACE] /Users/Matthew/Titanium/_Playground/Catalyst/build/iphone/Catalyst-Bridging-Header.h:6:9: note: in file included from /Users/Matthew/Titanium/_Playground/Catalyst/build/iphone/Catalyst-Bridging-Header.h:6:
[TRACE] #import "TiAppiOSActivityAttributesProxy.h"
[TRACE]         ^
[TRACE] 2 errors generated.
[ERROR] --- xcodebuild: WARNING: Using the first of multiple matching destinations:
[ERROR] { platform:macOS, arch:arm64, variant:Mac Catalyst, id:00006001-001210200A02801E }
[ERROR] { platform:macOS, arch:x86_64, variant:Mac Catalyst, id:00006001-001210200A02801E }
[ERROR] { platform:macOS, arch:arm64, variant:Designed for [iPad,iPhone], id:00006001-001210200A02801E }
[ERROR] { platform:iOS, id:dvtdevice-DVTiPhonePlaceholder-iphoneos:placeholder, name:Any iOS Device }
[ERROR] { platform:iOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-iphonesimulator:placeholder, name:Any iOS Simulator Device }
[ERROR] { platform:macOS, variant:Mac Catalyst, name:Any Mac }
[ERROR] { platform:iOS Simulator, id:FB7CF4EE-69A4-4243-A364-B9E25C721ED8, OS:13.7, name:iPad (7th generation) }
[ERROR] { platform:iOS Simulator, id:F3BB5D71-D687-4AF1-9119-4B9F32CA164A, OS:14.0.1, name:iPad (8th generation) }
[ERROR] { platform:iOS Simulator, id:6337A94E-05F9-420E-8EC5-EB636E502281, OS:16.2, name:iPad (9th generation) }
[ERROR] { platform:iOS Simulator, id:67FF7672-5129-4B81-BEC7-A1869E52FA84, OS:16.2, name:iPad (10th generation) }
[ERROR] { platform:iOS Simulator, id:117B5DD8-88DB-46FF-A51F-1CB357BD831B, OS:13.7, name:iPad Air (3rd generation) }
[ERROR] { platform:iOS Simulator, id:B69051CA-5C1E-45F4-9B2E-597BF69A20C1, OS:14.0.1, name:iPad Air (4th generation) }
[ERROR] { platform:iOS Simulator, id:6B436BC5-5D22-4C0E-BE4D-2A9E65DC408A, OS:16.2, name:iPad Air (5th generation) }
[ERROR] { platform:iOS Simulator, id:8B1F1A06-558F-466F-A5F9-2460F0DE2BC8, OS:13.7, name:iPad Pro (9.7-inch) }
[ERROR] { platform:iOS Simulator, id:1B0F05D4-E3D3-4F4F-A3B7-42A82338F0F6, OS:14.0.1, name:iPad Pro (9.7-inch) }
[ERROR] { platform:iOS Simulator, id:19C87224-7DA5-42CE-84ED-A391E2961351, OS:13.7, name:iPad Pro (11-inch) (2nd generation) }
[ERROR] { platform:iOS Simulator, id:B2255D47-265F-4ADE-8179-9B749A82830D, OS:14.0.1, name:iPad Pro (11-inch) (2nd generation) }
[ERROR] { platform:iOS Simulator, id:E96DF372-22FE-41CA-8FA1-A3A81E828CA3, OS:16.2, name:iPad Pro (11-inch) (3rd generation) }
[ERROR] { platform:iOS Simulator, id:07185221-FB1C-4BB9-A227-BECDBB8D88F5, OS:16.2, name:iPad Pro (11-inch) (4th generation) }
[ERROR] { platform:iOS Simulator, id:4820E46D-31F1-4034-A40F-ADE2DE9A10E2, OS:16.2, name:iPad Pro (12.9-inch) (2nd generation) }
[ERROR] { platform:iOS Simulator, id:8D99AEC2-3574-44EB-BF1C-433BC46F6BC0, OS:13.7, name:iPad Pro (12.9-inch) (4th generation) }
[ERROR] { platform:iOS Simulator, id:E828B35D-3784-4666-8318-93E311001CA3, OS:14.0.1, name:iPad Pro (12.9-inch) (4th generation) }
[ERROR] { platform:iOS Simulator, id:3DB35094-A636-458A-AD77-6A197B240028, OS:16.2, name:iPad Pro (12.9-inch) (5th generation) }
[ERROR] { platform:iOS Simulator, id:3B03311B-E9BF-4C2D-8CB4-5366C42FD657, OS:16.2, name:iPad Pro (12.9-inch) (6th generation) }
[ERROR] { platform:iOS Simulator, id:368C1487-2D0A-46D3-B2A2-E44556004ED9, OS:16.2, name:iPad mini (6th generation) }
[ERROR] { platform:iOS Simulator, id:C182E4AE-0D41-48A6-AFD5-E94C82ACF3BB, OS:16.2, name:iPhone 8 Plus }
[ERROR] { platform:iOS Simulator, id:1250CB23-726E-4379-A72F-90D40D288150, OS:13.7, name:iPhone 11 }
[ERROR] { platform:iOS Simulator, id:DF89A504-7B54-4AC2-BFFD-5142699F8FBE, OS:13.7, name:iPhone 11 Pro }
[ERROR] { platform:iOS Simulator, id:D22D8C72-E329-4B24-826E-94448F7622C7, OS:13.7, name:iPhone 11 Pro Max }
[ERROR] { platform:iOS Simulator, id:EBFF6DFD-C1F0-4763-BB81-8F23C7C9DB2F, OS:14.0.1, name:iPhone 11 Pro Max }
[ERROR] { platform:iOS Simulator, id:7D06A0B0-00B9-48B2-A06A-8A71F45F2351, OS:16.2, name:iPhone 11 Pro Max }
[ERROR] { platform:iOS Simulator, id:A58FF29E-5ACA-4C80-9A3A-DB267A9B28CA, OS:16.2, name:iPhone 14 }
[ERROR] { platform:iOS Simulator, id:43EE5E0B-727F-41A1-A8DD-481FA73DA6A4, OS:16.0, name:iPhone 14 Plus }
[ERROR] { platform:iOS Simulator, id:D0070ACC-583A-433D-9E7E-B261D09E3FC9, OS:16.2, name:iPhone 14 Plus }
[ERROR] { platform:iOS Simulator, id:5B58CEE2-1392-46B8-848D-C632E917A092, OS:16.2, name:iPhone 14 Pro }
[ERROR] { platform:iOS Simulator, id:38F5166F-529D-4589-A08E-1BBCBAD51E4E, OS:16.2, name:iPhone 14 Pro Max }
[ERROR] { platform:iOS Simulator, id:E211DE98-7430-4EAE-BE09-A8A42E7AE03D, OS:13.7, name:iPhone SE (2nd generation) }
[ERROR] { platform:iOS Simulator, id:1E5B00DB-F6D6-42AF-9977-E707D91BDB96, OS:14.0.1, name:iPhone SE (2nd generation) }
[ERROR] { platform:iOS Simulator, id:435BA1CD-24AE-423E-889F-61AFE60A64E5, OS:16.2, name:iPhone SE (3rd generation) }
[ERROR] { platform:iOS Simulator, id:A208F81F-B282-4B41-BF37-83E36F9D0380, OS:16.2, name:iPhone Xs }
[ERROR] { platform:iOS Simulator, id:86229C6C-D4D0-45B5-9892-44BD3CF30C57, OS:14.0.1, name:iPod touch (7th generation) }
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEPLOYTYPE=development')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEPLOYTYPE=development')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than '__LOG__ID__=1151ae93-c29b-4c45-b1c7-8ce2637879b6')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than '__LOG__ID__=1151ae93-c29b-4c45-b1c7-8ce2637879b6')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEBUG=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEBUG=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'LAUNCHSCREEN_STORYBOARD=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'LAUNCHSCREEN_STORYBOARD=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEFAULT_BGCOLOR_RED=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEFAULT_BGCOLOR_RED=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEFAULT_BGCOLOR_GREEN=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEFAULT_BGCOLOR_GREEN=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEFAULT_BGCOLOR_BLUE=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DEFAULT_BGCOLOR_BLUE=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DISABLE_TI_LOG_SERVER=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'DISABLE_TI_LOG_SERVER=1')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'TI_VERSION=12.0.0')
[ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'TI_VERSION=12.0.0')
[ERROR] ** BUILD FAILED **
[ERROR] The following build commands failed:
[ERROR]         SwiftGeneratePch normal x86_64 Compiling\ bridging\ header (in target 'Catalyst' from project 'Catalyst')
[ERROR]         SwiftGeneratePch normal arm64 Compiling\ bridging\ header (in target 'Catalyst' from project 'Catalyst')
[ERROR] (2 failures)
```

--- a/.github/actions/build-ios/action.yml
+++ b/.github/actions/build-ios/action.yml
@@ -9,7 +9,7 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Use Node.js 12.x
+    - name: Use Node.js 16.x
       uses: actions/setup-node@v2
       with:
         node-version: '16.x'

--- a/.github/actions/package/action.yml
+++ b/.github/actions/package/action.yml
@@ -15,7 +15,7 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Use Node.js 12.x
+    - name: Use Node.js 16.x
       uses: actions/setup-node@v2
       with:
         node-version: ${{ inputs.node-version }}

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -25,12 +25,12 @@ jobs:
           java-version: '11'
 
   ios:
-    runs-on: macos-11
+    runs-on: macos-12
     name: iOS
     env:
       CCACHE_DIR: ${{ github.workspace }}/.ccache
       USE_CCACHE: 1
-      DEVELOPER_DIR: /Applications/Xcode_13.2.1.app/Contents/Developer
+      DEVELOPER_DIR: /Applications/Xcode_14.2.app/Contents/Developer
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
@@ -50,7 +50,7 @@ jobs:
       with:
         fetch-depth: 0
 
-    - name: Use Node.js 12.x
+    - name: Use Node.js 16.x
       uses: actions/setup-node@v2
       with:
         node-version: '16.x'
@@ -63,11 +63,11 @@ jobs:
       run: npm run lint:js
   
   package:
-    runs-on: macos-11
+    runs-on: macos-12
     name: Package
     env:
       SDK_BUILD_CACHE_DIR: ${{ github.workspace }}/.native-modules
-      DEVELOPER_DIR: /Applications/Xcode_13.2.1.app/Contents/Developer
+      DEVELOPER_DIR: /Applications/Xcode_14.2.app/Contents/Developer
     needs: [android, ios, js]
     steps:
     - name: Checkout repository

--- a/.github/workflows/docs.yml
+++ b/.github/workflows/docs.yml
@@ -16,7 +16,7 @@ jobs:
         uses: actions/checkout@v3
         with:
             fetch-depth: 0
-      - name: Use Node.js 12.x
+      - name: Use Node.js 16.x
         uses: actions/setup-node@v2
         with:
           node-version: '16.x'

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -75,13 +75,13 @@ jobs:
           java-version: '11'
 
   ios:
-    runs-on: macos-11
+    runs-on: macos-12
     name: iOS
     needs: [validate]
     env:
       CCACHE_DIR: ${{ github.workspace }}/.ccache
       USE_CCACHE: 1
-      DEVELOPER_DIR: /Applications/Xcode_13.2.1.app/Contents/Developer
+      DEVELOPER_DIR: /Applications/Xcode_14.2.app/Contents/Developer
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
@@ -94,11 +94,11 @@ jobs:
           node-version: '16.x'
 
   package:
-    runs-on: macos-11
+    runs-on: macos-12
     name: Package
     env:
       SDK_BUILD_CACHE_DIR: ${{ github.workspace }}/.native-modules
-      DEVELOPER_DIR: /Applications/Xcode_13.2.1.app/Contents/Developer
+      DEVELOPER_DIR: /Applications/Xcode_14.2.app/Contents/Developer
       vtag: ${{ needs.validate.outputs.vtag }}
     needs: [validate, android, ios]
     steps:

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -128,7 +128,7 @@ def androidUnitTests(testName, nodeVersion, npmVersion, deviceId) {
 
 def macosUnitTests(nodeVersion, npmVersion) {
 	return {
-		node('git && xcode-13') {
+		node('git && xcode-14') {
 			// TODO: Do a shallow checkout rather than stash/unstash?
 			unstash 'mocha-tests'
 			try {
@@ -165,7 +165,7 @@ def macosUnitTests(nodeVersion, npmVersion) {
 
 def iosUnitTests(deviceFamily, nodeVersion, npmVersion) {
 	return {
-		node('git && xcode-13') {
+		node('git && xcode-14') {
 			// TODO: Do a shallow checkout rather than stash/unstash?
 			unstash 'mocha-tests'
 			try {
@@ -225,7 +225,7 @@ def cliUnitTests(nodeVersion, npmVersion) {
 // Wrap in timestamper
 timestamps {
 	try {
-		node('git && android-sdk && gperf && xcode-13') {
+		node('git && android-sdk && gperf && xcode-14') {
 			env.JAVA_HOME="${tool name:'OpenJDK 11.0.11+9', type: 'jdk'}"
 			env.PATH="${env.JAVA_HOME}/bin:${env.PATH}"
 

